cfme/tests/control/test_alerts.py

```diff
@@ -341,7 +341,7 @@ def test_alert_timeline_cpu(request, vm, set_performance_capture_threshold, prov
     timeline = vm.open_timelines()
     timeline.filter.fill({
         "event_category": "Alarm/Status Change/Errors",
-        "time_range": "Days",
+        "time_range": "Weeks",
```

Comment:
**Question**:  Why we need to change it to 'Weeks'?

Reply:
@rlbabyuk No idea, when I was refactoring this test last time it worked with "Days". I guess timelines behavior has changed and now in this test the event can be found only with "Weeks".